src/main/scala/no/ndla/imageapi/service/WriteService.scala

```diff
@@ -160,14 +160,35 @@ trait WriteService {
         )
     }
 
-    def updateImage(imageId: Long, image: UpdateImageMetaInformation): Try[ImageMetaInformationV2] = {
-      val oldImage = imageRepository.withId(imageId)
-      val imageToUpdateWith = oldImage match {
-        case None           => Failure(new ImageNotFoundException(s"Image with id $imageId found"))
-        case Some(existing) => Success(mergeImages(existing, image))
-      }
+    private def overwriteImage(newFile: FileItem, oldImage: ImageMetaInformation): Try[ImageMetaInformation] = {
+      uploadImage(newFile).map(uploaded => converterService.withNewImage(oldImage, uploaded))
```

Comment:
Jeg vurderte å hive inn en slett gamle bildet og redirect her, vil vi det? 

Da brekker vi potensielle direktelenker som er brukt av "andre".

Jeg tenkte først på en redirect, men det var ikke så lett å få til med s3 som jeg hadde håpt.
Om vi skal gjøre det så må vi legge på en header på det gamle bildet og lese den i api-gateway (og image-api) for så å gjøre en redirect. Kanskje dyrt?

Reply:
Poenget her er å oppdatere bildet i ny og bedre versjon, så egentlig bør gamle lenker fungere som før, berre peikende på nytt bilde.

Reply:
På NDLA sidene så blir jo nytt bilde returnert fra apiet, men direktelenker er litt verre å løse. Kan undersøke litt nøyere hvor mye jobb det er å løse i api-gateway.